scripts/dockerfiles/run.sh

```diff
@@ -85,6 +85,52 @@ done
         /var/vcap/jobs/{{ $job.Name }}/bin/run
     {{ end }}
 {{ else }}
+    # Run all the scripts called post-start, if any.
+    function post-start-paths()
+    {
+	declare -a fnames
+	idx=0
+	for fname in $(find /var/vcap/jobs/*/bin -name post-start) ; do
+	    fnames[$idx]=$fname
+	    idx=$((idx + 1))
+	done
+	echo ${fnames[*]}
+    }
+
+    psp=$(post-start-paths)
+
+    if [ "X$psp" != X ] ; then
+	# We have post-start scripts to run.
+	# We
+	# - Start a transient monit in the background, treat it as
+	#   non-background however
+	# - Wait for all jobs (except `ready`) to be ok.
+	# - Run the post-start scripts.
+	# - Get rid of the transient monit
+	# - Mark job `ready` as ok, and start the final monit,
+	#   replacing ourselves.
+	# The last two operations are also done when we have no
+	# post-start scripts.
+
+	dumb-init -- monit -vI &
+	sleep 10
+	while [ $(monit report up) -gt 1 ] ; do
```

Comment:
The comment above says:

> Wait for all jobs (except `ready`) to be ok.

This line seems to wait for just the first job to start.